web-app/src/components/BookingModal/LegacyMessageList/styled.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+
+export const Ul = styled.ul`
+    li {
+        list-style-type: none;
+        margin: 10px 10px 10px 10px;
+        border-radius: 25px;
+        width: 75%;
+        padding-left: 30px;
+        padding-top: 1px;
+        padding-bottom: 1px;
+        word-wrap:break-word;
+    }
+
+    h1 {
+        margin: 10px 10px 10px 10px;
+        position: relative;
+        color:white;
+    }
+
+    h3 {
+        text-transform: none;
+        margin: 7px 10px 13px 7px;
+        color: white;
+        font-family: Open Sans,sans-serif;
+        padding-right: 20px;
+        padding-left: 19px;
```

Comment:
Quite pedantic here but it's been drilled into me, how come one is 20px and one is 19px?